tests/new_api/test_serialized_links.py

```diff
@@ -21,95 +21,98 @@
 def draft_json(client, minimal_input_record, es):
     """Bibliographic Draft fixture."""
     response = client.post(
-        "/rdm-records", json=minimal_input_record, headers=HEADERS
+        "/records", json=minimal_input_record, headers=HEADERS
     )
     return response.json
 
 
 @pytest.fixture
-def published_json(client, minimal_input_record, es):
+def published_json(app, client, minimal_input_record, es):
     """Bibliographic Record fixture.
 
     Can't depend on draft_json since publication deletes draft.
     """
     response = client.post(
-        "/rdm-records", json=minimal_input_record, headers=HEADERS
+        "/records", json=minimal_input_record, headers=HEADERS
     )
-    pid_value = response.json["pid"]
+    pid_value = response.json["id"]
     response = client.post(
-        f"/rdm-records/{pid_value}/draft/actions/publish", headers=HEADERS
+        f"/records/{pid_value}/draft/actions/publish", headers=HEADERS
     )
     return response.json
 
 
+@pytest.mark.skip()
```

Comment:
@fenekku I skipped this one because I have changed the resource/service configuration! You can check the `tests/resources` module. Maybe it's worth to transfer these tests there and slowly review the old ones and keep only the ones that make sense :)